### PR TITLE
started animations

### PR DESCRIPTION
**Pull request by Drex72**

### CODE REVIEW

File: src/styles/index.scss
Comment (TemitopeAsama) [inline src/styles/index.scss:1]: "disappear" is spelt wrong.

File: src/components/home/landing-section/landing.tsx
Comment (TemitopeAsama) [inline src/components/home/landing-section/landing.tsx:1]: Again, the RevealAnimation wrapper thing might not be the best approach to this implementation. Do you need help with figuring out better alternatives?

**Review by TemitopeAsama — CHANGES_REQUESTED** [review]
Fantastic work all in all. Love the speed and progress. Kindly implement feedback and let's have a conversation about the wrapper thing. Thank you.

**Comment by Drex72** [comment]
> Fantastic work all in all. Love the speed and progress. Kindly implement feedback and let's have a conversation about the wrapper thing. Thank you.

Alright
Thanks boss

**Pull request closed without merging**